### PR TITLE
897 update docs

### LINKED ISSUE #897: Update screen shots in user guide
### Desired Behaviour

Some of the GUI has changed a lot. Make sure the images in the docs are close enough to current to be useful.

### PR DESCRIPTION
**Pull request by samtygier-stfc**
### Issue
Closes #897 

### Description

Update screen shots and documentation

### Testing 

`python setup.py docs`

### Acceptance Criteria 

Screenshots and text are up to date

### Documentation

Doc change only

### CODE REVIEW

File: docs/user_guide/gui/loading_saving.rst [hunk 1]
```diff
@@ -1,44 +1,33 @@
...
+When *Load dataset* is selected, the following dialog will appear:
 
-.. image:: ../../_static/gui_load_dialog_empty.png
+.. image:: ../../_static/loading_screen.png
     :alt: Initial Load dialog
 
-This allows you to select the data you wish to load; in the case of TIFF or FITS
-images then any image in a directory containing an image stack can be selected
-or a single NeXus file if loading from NeXus.
+This allows you to select the data you wish to load; The Sample entry should be used to select a the first projection TIFF file, the remaining TIFF files in the directory will be automatically detected if their filename matches the pattern. The loading dialog will try to automatically find associated files including flat fields, dark frames, 180 degree projections and sample logs.
```
Comment (DolicaAkelloEgwel) [inline docs/user_guide/gui/loading_saving.rst:15]:
```suggestion
This allows you to select the data you wish to load; The Sample entry should be used to select the first projection TIFF file, the remaining TIFF files in the directory will be automatically detected if their filename matches the pattern. The loading dialog will try to automatically find associated files including flat fields, dark frames, 180 degree projections and sample logs.
```

File: docs/user_guide/gui/loading_saving.rst [hunk 2]
```diff
@@ -1,44 +1,33 @@
...
 
 Once data has been selected, it's shape is inspected and the stack index fields
```
Comment (DolicaAkelloEgwel) [inline docs/user_guide/gui/loading_saving.rst:17]:
```suggestion
Once data has been selected, its shape is inspected and the stack index fields
```

File: docs/user_guide/gui/loading_saving.rst [hunk 3]
```diff
@@ -1,44 +1,33 @@
...
 pre populated as shown below:
```
Comment (DolicaAkelloEgwel) [inline docs/user_guide/gui/loading_saving.rst:18]:
```suggestion
are populated as shown below:
```

File: docs/user_guide/gui/loading_saving.rst [hunk 4]
```diff
@@ -58,6 +47,3 @@ of the image in the stack.
 The *Overwrite* option removes checking for existing data of the same name. When
 disabled an error will be raised of the specified output directory is not empty
```
Comment (DolicaAkelloEgwel) [inline docs/user_guide/gui/loading_saving.rst:48]:
```suggestion
disabled an error will be raised if the specified output directory is not empty
```

File: docs/user_guide/gui/operations_window.rst [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+=================
+
+Selecting the *Operations* option in the *Workflow* menu opens the operations
+dialog, this is used for applying filters to image stacks.
+
+Operations may be applied to image stacks of any type (i.e. projection, sinogram,
+reconstruction), they are essentially just image filters and do not directly
+care about the type of image.
+
+.. image:: ../../_static/operations_window.png
```
Comment (DolicaAkelloEgwel) [inline docs/user_guide/gui/operations_window.rst:11]: Can't find this image.

File: docs/user_guide/gui/operations_window.rst [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+    :alt: Operations dialog
+    :width: 90%
+
+Here you can select the type of filter and the image stack you wish to apply it
+to, properties of the selected filter are automatically shown in the *Filter
+Properties* section.
+
+The [?] button next to filter selector will open a webpage will an explanation of the filter parameters.
```
Comment (DolicaAkelloEgwel) [inline docs/user_guide/gui/operations_window.rst:19]:
```suggestion
The [?] button next to filter selector will open a webpage with an explanation of the filter parameters.
```

File: docs/user_guide/gui/operations_window.rst [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+
+The right hand panel shows a slice from the original image stack, a preview of the filter applied to the slice, and the pixel intensity difference. Below a histogram of pixel values before and after is shown. The image views can be navigated as described in the :ref:`Image View` help page.
+
+There are several tickboxes to customised the behaviour of the preview area.
```
Comment (DolicaAkelloEgwel) [inline docs/user_guide/gui/operations_window.rst:23]:
```suggestion
There are several tickboxes to customise the behaviour of the preview area.
```

File: docs/user_guide/gui/operations_window.rst (hunk 5 above)
Comment (samtygier-stfc) [reply docs/user_guide/gui/operations_window.rst:11]: Added

**Review by DolicaAkelloEgwel — APPROVED** [review]

**Pull request merged**